packages/cfd/src/Containers/cfd-server-error-dialog.tsx

```diff
@@ -1,35 +1,20 @@
 import React from 'react';
 import { Dialog } from '@deriv/components';
 import { localize, Localize } from '@deriv/translations';
-import { connect } from '../Stores/connect';
-import RootStore from '../Stores/index';
+import { useStore } from '@deriv/stores';
 
-type TCFDServerErrorDialogProps = {
-    clearCFDError: () => void;
-    context: RootStore;
-    disableApp: () => void;
-    enableApp: () => void;
-    error_message: string;
-    error_type?: string;
-    has_cfd_error: boolean;
-    is_cfd_success_dialog_enabled: boolean;
-};
+const CFDServerErrorDialog = () => {
```

Comment:
Refactored the `CFDServerErrorDialog` component to remove the `connect` method and replace it with the `useStore` hook from `@deriv/stores` as an example.